queries.sql

```diff
@@ -59,4 +59,34 @@ GROUP BY species;
 SELECT species, AVG(escape_attempts) AS average_escape_attempts
 FROM animals
 WHERE date_of_birth BETWEEN 'jan-1-1990' AND 'jan-1-2000'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+/*querying  tables using join */
```

Comment:
- [ ] `README.md` file: To make your documentation looks more professional, kindly ensure that you update your README sections with appropriate information and for those without information, I can suggest you remove them from now and add them later if you find information to fill in. For example:
  - Add a good description of your project.
![Screenshot from 2023-07-14 11-39-37](https://github.com/Shaheryar0054/vet_clinic/assets/83042742/641eaf13-a1ba-4113-a1de-96d68c658861)  
  - Add appropriate information to the `Show your support` section
![Screenshot from 2023-07-14 11-39-54](https://github.com/Shaheryar0054/vet_clinic/assets/83042742/dcca37c5-e3a0-471c-b93e-8f246a250307)
  - Add appropriate information to below section or remove those which don't have information related to your project
![Screenshot from 2023-07-14 11-37-37](https://github.com/Shaheryar0054/vet_clinic/assets/83042742/5a985df2-e423-4fa8-8d61-9858b3e2e946)


- [ ] I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same content in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**
![Screenshot from 2023-07-14 11-32-35](https://github.com/Shaheryar0054/vet_clinic/assets/83042742/a04c8855-f120-4faa-930e-dfc122815042)